llvm/tools/llvm-reduce/deltas/ReduceMetadata.cpp

```diff
@@ -28,40 +28,24 @@ static bool shouldKeepDebugNamedMetadata(NamedMDNode &MD) {
   return MD.getName() == "llvm.dbg.cu" && MD.getNumOperands() != 0;
 }
 
-// Named metadata with simple list-like behavior, so that it's valid to remove
-// operands individually.
```

Comment:
We should preserve this. If we want a mode where we blindly rip operands out of anything we don't understand, probably should be a flag